versions.yaml

```diff
@@ -26,13 +26,14 @@ versions:
     templateSubDir: '8'
     templateArgs:
       exporter: 'true'
+      c2dRelease: '8.0.0'
     tags:
       - '8.0-debian9'
       - '8-debian9'
       - '8.0'
       - '8'
       - 'latest'
-    from: *from
+    from: gcr.io/google-appengine/debian9
```

Comment:
Revert to `*from` or remove `from` from `_variables`.

Reply:
Newer versions have different "from". Removed from variables and added explicit from to older versions.